# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/html/HTMLInputElement.h
# Commit: b9e87fde31f7
# Full Hash: b9e87fde31f7ca31185081dc96fa937927f1879f
# Author: sefeng <sefeng@mozilla.com>
# Date: 2020-05-12 04:04:10
# Regressor Bug: 1557303
# File Overlap Count: 1
# Description:
#   Bug 1557303 - Implement form[method="dialog"] r=smaug
#   
#   Spec: https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#submit-dialog
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D73364
# ==============================================================================

diff -r e02018dfce9e -r b9e87fde31f7 dom/html/HTMLInputElement.h
--- a/dom/html/HTMLInputElement.h	Tue May 12 00:11:12 2020 +0000
+++ b/dom/html/HTMLInputElement.h	Mon May 11 22:56:13 2020 +0000
@@ -962,6 +962,8 @@
 
   virtual void AfterClearForm(bool aUnbindOrDelete) override;
 
+  virtual void ResultForDialogSubmit(nsAString& aResult) override;
+
   /**
    * Dispatch a select event. Returns true if the event was not cancelled.
    */